server/middleware/setUpCookies.ts

```diff
@@ -11,6 +11,13 @@ export default function setUpCookies(): Router {
     next()
   })
 
+  router.get('/cookie-policy', (req, res) => {
+    res.render('pages/cookiePolicy', {
+      pageHeading: 'Cookies',
+      preference: req.cookies?.acceptAnalyticsCookies,
```

Comment:
Won't we have this from `res.locals.acceptAnalyticsCookies = req.cookies?.acceptAnalyticsCookies` above?